### PR TITLE
AMP-629 update spec for private beta

### PR DESCRIPTION
**Pull request by tonyheap**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/patient-care-aggregator-api.yaml [hunk 1]
```diff
@@ -92,7 +91,7 @@ info:
     | ----------------- | ----------------------------------------------------------------------------- | ----------------------------- |
     | Sandbox           | Not available                                                                 | Not applicable                |
     | Integration test  | `https://int.api.service.nhs.uk/patient-care-aggregator-api/FHIR/R4/`         | NHS login - integration (AOS) |
-    | Production        | Not yet available                                                             | NHS login - Live (production) |
+    | Production        | `https://api.service.nhs.uk/patient-care-aggregator-api/FHIR/R4/`             | NHS login - Live (production) |
```
Comment (deskelleher) [inline specification/patient-care-aggregator-api.yaml:94]: The production URL is https://api.service.nhs.uk/patient-care-aggregator-api/aggregator/events. The change to /FHIR/R4 is on our backlog and will happen after we have gone live with the service as a fast follow

Comment (tonyheap) [reply specification/patient-care-aggregator-api.yaml]: Changed

File: specification/patient-care-aggregator-api.yaml [hunk 2]
```diff
@@ -92,7 +91,7 @@ info:
     | ----------------- | ----------------------------------------------------------------------------- | ----------------------------- |
     | Sandbox           | Not available                                                                 | Not applicable                |
     | Integration test  | `https://int.api.service.nhs.uk/patient-care-aggregator-api/FHIR/R4/`         | NHS login - integration (AOS) |
```
Comment (deskelleher) [inline specification/patient-care-aggregator-api.yaml:93]: The URL for INT is https://int.api.service.nhs.uk/patient-care-aggregator-api/events

Comment (tonyheap) [reply specification/patient-care-aggregator-api.yaml:93]: Do you mean /events or do you mean /aggregator/events, as per prod?

Comment (deskelleher) [reply specification/patient-care-aggregator-api.yaml:93]: You're right, it's https://int.api.service.nhs.uk/patient-care-aggregator-api/aggregator/events

Comment (tonyheap) [reply specification/patient-care-aggregator-api.yaml]: Fixed

**Review by deskelleher — APPROVED** [review]

**Pull request merged**